pkg/frequencyplans/frequencyplans.go

```diff
@@ -412,6 +414,8 @@ func (fp FrequencyPlan) Extend(extension FrequencyPlan) FrequencyPlan {
 		val := *extension.MaxEIRP
 		extended.MaxEIRP = &val
 	}
+	extended.Gateways = extension.Gateways
```

Comment:
@KrishnaIyer currently always using the `Gateways` value coming from the file and not extending it. So if the base file is applicable for the gateway (gateways=true) the inherited file can overwrite it (gateways=false), otherwise the inherited file cannot overwrite the base file value if it just extending it.

Reply:
If you use `*bool` for the field, we can distinguish between the field being explicitly set to false and the field being omitted.
So I think we should override this only if it's explicitly set in the extension.

If there is a case where the base plan is applicable for gateways and the extension plan should not be used for gateways but only for end devices, then we can explicitly set it to false in the extension and that can then override the base.

Reply:
Good idea, I've changed it.